### ISSUE TITLE
Grouped product label is wrong in single product 

### ISSUE THREAD

**Issue by profictions** [woocommerce/woocommerce#15046]
### EXPLANATION OF THE ISSUE

Under WC 3.0.6, /plugins/woocommerce/templates/single-product/add-to-cart/grouped.php is wrong on line 70. 

It reads:

`<?php echo $grouped_product->is_visible() ? '<a href="' . esc_url( apply_filters( 'woocommerce_grouped_product_list_link', get_permalink(), $grouped_product->get_id() ) ) . '">' . get_the_title() . '</a>' : get_the_title(); ?>`

but should read:

`<?php echo $grouped_product->is_visible() ? '<a href="' . esc_url( apply_filters( 'woocommerce_grouped_product_list_link', get_permalink(), $grouped_product->get_id() ) ) . '">' . $grouped_product->get_title() . '</a>' : $grouped_product->get_title(); ?>`

### SYSTEM STATUS

<details>
```
Grab the system status report from WooCommerce > System Status and paste it here between the `details` tags.
```
</details>

**Comment by mikejolley** [comment]
This is fixed in master/3.0.7

**Comment by profictions** [comment]
Thanks!
